packages/backend/src/logger/config.d.ts

```diff
@@ -0,0 +1,85 @@
+export interface Config {
+  /**
+   * Configuration for the audit logging
+   * @visibility frontend
+   */
+  auditLog?: {
+    /**
+     * Configuration for the audit logging to console
+     * @visibility frontend
+     */
+    logToConsole: {
```

Comment:
```suggestion
    console: {
```
I think we should rename this.